src/unix/fsevents.c

```diff
@@ -649,9 +649,15 @@ static int uv__fsevents_loop_init(uv_loop_t* loop) {
   if (pthread_attr_init(attr))
     attr = NULL;
 
+  /**
+   * macOS 10.15 FSEvents uses alloca() stack allocations, requiring a larger
+   * stack size. Leave this as the default 512KB size in that case.
+   */
+  #if MAC_OS_X_VERSION_MAX_ALLOWED < 101500
```

Comment:
Not sure if it's necessary to gate this change to macOS 10.15+. I'd personally prefer to remove the `pthread_attr_setstacksize` call entirely, but curious for thoughts from reviewers.

Reply:
I agree

Reply:
Thanks for taking a look. I went through route for the latest push. May revert depending on the discussion below. https://github.com/libuv/libuv/pull/3132#discussion_r592866512